Packages/com.unity.inputsystem/InputSystem/Plugins/PlayerInput/PlayerInput.cs

```diff
@@ -629,6 +629,8 @@ private static PlayerInput DoInstantiate(GameObject prefab)
         // by the receiver. We reuse the same object over and over to avoid allocating garbage.
         [NonSerialized] private InputValue m_InputValueObject;
 
+        [NonSerialized] internal InputActionMap m_EnabledActionMap;
```

Comment:
Do we want to expose this? Maybe a `currentActionMap` property? The way this works now makes me think we have the concept of a "current" action map now. "Default Action Map" is the one that becomes current by default and `SwitchActions` (which I think should then probably be called `SwitchCurrentActionMap`) changes the currently active one.

Reply:
I'm not sure about renaming `SwitchCurrentActionMap`. We have a test called `PlayerInput_CanSwitchActionsWithMessage` which shows behavior which hardcodes the name of this function as a string, so I'm scared to change that. But I'd like to understand: what's the use case of this? It will likely cause problems on il2cpp with stripping. Also, should we keep `SwitchActions` at all? The `currentActionMap` is settable, making it redundant (but maybe still convenient  because it uses the action map name?).